src/sankoff_handler.hpp

```diff
@@ -99,45 +126,51 @@ TEST_CASE("SankoffHandler: Tests on single site sequence.") {
       RootedTreeCollection::OfTreeCollection(driver.ParseNewickFile(newick_file));
   SitePattern site_pattern = SitePattern(alignment, tree_collection.TagTaxonMap());
   Node::NodePtr topology = tree_collection.GetTree(0).Topology();
+  size_t taxon_count = site_pattern.TaxonCount();
 
   // transitions have cost 1 and transversions have cost 2.5
   auto costs = CostMatrix();
   costs << 0., 2.5, 1., 2.5,  //
       2.5, 0., 2.5, 1.,       //
       1., 2.5, 0., 2.5,       //
       2.5, 1., 2.5, 0.;       //
+
```

Comment:
My guess is that things are fine, but it might be nice to have a less "symmetric" test. You might have to ask Will D for help running Sankoff with a different matrix.

Reply:
Did you want to do this, @marybarker ?

Reply:
Ah, yes! That is on my list and I forgot. I'll check with Will D

Reply:
there is now a test case with an asymmetric cost matrix.